live_debug_view/templates/live_debug_view/base.html

```diff
@@ -0,0 +1,54 @@
+{% load static %}
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>{% block title %}{% endblock title %}</title>
+    <link href="{% static 'live_debug_view/css/main.css' %}" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap" rel="stylesheet"> 
+  </head>
+  <body>
+    <div class="wrapper">
+      <header>
+        {% block header %}{% endblock header %}
+      </header>
+
+      <div class="infos">
+        <div id="success"></div>
+        <div id="error"></div>
+      </div>
+      <main>
+        <section class="compose-form">
+          {% block form %}{% endblock form %}
+        </section>
+        <section class="content">
+          {% block content %}{% endblock content %}
+        </section>
+      </main>
+    </div>
+
+    <script>
+      function showNotif(msg, reason=1){
+          if (reason == 1){
```

Comment:
Doesn't matter in this case, but be careful that in Javascript stuff like `'1' == 1` evaluates to true, whereas `'1' === 1` will evaluate to false.

Reply:
Okay, I will pay attention.